src/App.jsx

```diff
@@ -1,14 +1,32 @@
 import './App.css';
+import messagesData from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import { useState } from 'react';
 
 const App = () => {
+  const [chatMessages, setChatMessages] = useState(messagesData);
+
+  const toggleLike = (id) => {
+    const updatedMessages = chatMessages.map((msg) => {
+      if (msg.id === id) {
+        return { ...msg, liked: !msg.liked };
+      }
+      return msg;
+    });
+
+    setChatMessages(updatedMessages);
+  };
+
+  const totalLikes = chatMessages.filter((msg) => msg.liked).length;
+
   return (
     <div id="App">
       <header>
-        <h1>Application title</h1>
+        <h1>Waiting for Godot</h1>
```

Comment:
Ha! I like that you put your own spin on this project, it's cute!


An element missing from your project is the header "Chat Between Vladimir and Estragon". Here you have "Waiting for Godot". 

@MarinaAdams How would you create a header that reflects the screenshot below? It can be done by hardcoding the string in the `h1` tag, but I'm wondering how would you programmatically get the names of the participants in the chat?

Please respond to this comment with your answer, thank you!

<img width="851" alt="image" src="https://github.com/user-attachments/assets/9425330e-3a71-4f6f-9471-e198f074805c" />